plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/EclipseProjectCreator.java

```diff
@@ -0,0 +1,21 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IWorkspaceRoot;
+import org.eclipse.core.resources.ResourcesPlugin;
+
+/**
+* Utility to make a new Eclipse project with the App Engine Standard Nature in the workspace.  
+*/
+class EclipseProjectCreator {
+
+  /**
+   * @return true on successful project creation; false otherwise
+   */
+  static boolean makeNewProject(AppEngineStandardProjectConfig config) {
```

Comment:
A boolean doesn't provide any information on failure.  There are a lot of reasons why project creation can go wrong.  Eclipse APIs return an IStatus object.  Or you could throw an exception.

This should take an `IProgressMonitor`; a dummy can be created with `NullProgressMonitor` (for tests).